docs/blueprints/EP031.rst

```diff
@@ -298,15 +296,44 @@ The **telemetry_int** napp must use a different cookie ID to help understanding
 XI. Consistency
 ===============
 
-The **telemetry_int** napp might deploy a routine to evaluate the consistency of the telemetry flows as performed by the **mef_eline** napp. This implementation will be defined via field experience with Kytos.
+The **telemetry_int** napp might deploy a routine to evaluate the consistency of the telemetry flows as performed by the **mef_eline** napp. This implementation will be defined via field experience with Kytos. Ideally, the consistency check should also rely on ``sdntrace_cp`` and follow the same pattern as ``mef_eline``, except that also when trying to trace, it should test both UDP and TCP payloads, if any fails, then it should eventually try to redeploy. As of ``sdntrace_cp`` version ``2023.1`` it still doesn't completely support ``goto_table`` neither ``instructions``, so it needs to be augmented just so ``telemetry_int`` can eventually also rely on it. In the meantime, once an telemetry INT EVC is deployed it'll be considered enabled, including after controller reload, which might not be the case, but since ``flow_manager`` also has a consistency eventually even if flows are accidentally deleted on switches, then it'll push missing flows again.
```

Comment:
We'll need to discuss if for production you'll need the consistency check completely implemented too, on `sdntrace_cp` these would also be needed:

https://github.com/kytos-ng/sdntrace_cp/issues/103

https://github.com/kytos-ng/sdntrace_cp/issues/104

Reply:
Yes, we will need to have consistency check completely implemented because otherwise, we can misclassify an EVC with INT enabled as working when it might not be fully functional.

Reply:
Additionally, the consistency should also check if the proxy port is active and take actions accordingly.

Reply:
> Yes, we will need to have consistency check completely implemented because otherwise, we can misclassify an EVC with INT enabled as working when it might not be fully functional.

Right. So, we'll need to include this extra effort on `sdntrace_cp` then. @gretelliz let me know if you could collaborate on that part and if you think it's feasible, this epic here has high priority for the `2023.2` version it doesn't need immediate action, but should be on the top of the list.

Reply:
> Additionally, the consistency should also check if the proxy port is active and take actions accordingly.

@italovalcy @jab1982, we'll need to specify here the expected behavior if the consistency check never succeeds, `mef_eline` worst case it to keep trying to redeploy, but `telemetry_int` in the worst case, should it remove the flows and give up to fallback to `mef_eline`? and then only try to activate/redeploy again if the EVC is active but also proxy ports are all set?

Reply:
Whatever the final decision on this related thread here https://github.com/kytos-ng/kytos/pull/402/files#r1276699359, it should probably follow the same behavior since ultimately `telemetry_int` is essentially "shadowing" `mef_eline` flows with higher priority ones and `mef_eline`'s is the one responsible for finding paths with the parametrized options.

Reply:
The main gotcha if it were to fall back and delete flows, is that consistency check wouldn't make sense from telemetry_int perspective to run again, since it'd be running the same consistency as mef_eline, and it can end up in a cycle in the worst case if a wrong action were to be installed. 

If a fallback to mef_eline is desirable when INT consistency check fails after a few attempts, then it should probably just disable INT and update status & status_reason metadata accordingly too. Consistency check from a INT perspective will only fail if flows haven't been installed yet or if indeed flows have wrong match or actions. So, as long as it doesn't pre-maturely use a short interval it should perform relatively stable. Mentioning these points just so you all are aware and we can make a decision of the behavior you need implemented.

Reply:
Final verdict: consistency check will disable INT (disable and remove the flows) if it fails after a few attempts in this first version. Updated the text on https://github.com/kytos-ng/kytos/pull/402/commits/c2330dce57198b2588b2b277402da8b9d7dde937